src/main/java/it/pn/frontend/e2e/rest/RestNotification.java

```diff
@@ -0,0 +1,39 @@
+package it.pn.frontend.e2e.rest;// ... Altre importazioni ...
+
+import it.pn.frontend.e2e.config.CustomHttpClient;
+import it.pn.frontend.e2e.exceptions.RestNotificationException;
+import it.pn.frontend.e2e.model.NewNotification;
+import it.pn.frontend.e2e.model.NewNotificationResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class RestNotification {
+    private static final Logger logger = LoggerFactory.getLogger("RestNotification");
+
+    public RestNotification() {
+    }
+
+    /**
+     * Create a new notification with one recipient and one document
+     *
+     * @param notification NewNotification object with all the data
+     * @return NewNotificationResponse object with the response
+     * @throws RestNotificationException if there is an error during the request
+     */
+    public NewNotificationResponse newNotificationWithOneRecipientAndDocument(NewNotification notification) {
```

Comment:
what is sent differently for a multi in the notification? (if not an array of recipients), change name if you think like me